### PR TITLE
Update [t-w]* from monorail results

### PR DESCRIPTION
**Pull request by stephenmcgruer**
See #798 for the methdology

### CODE REVIEW

File: webrtc-extensions/META.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+links:
+    - product: chrome
+      url: https://crbug.com/1044901
```
Comment (stephenmcgruer) [inline webrtc-extensions/META.yml:3]: Unclear if related

File: webrtc-priority/META.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+links:
+    - product: chrome
+      url: https://crbug.com/1084006
```
Comment (stephenmcgruer) [inline webrtc-priority/META.yml:3]: Unrelated

File: webrtc/META.yml [hunk 3]
```diff
@@ -67,8 +67,37 @@ links:
...
       url: https://crbug.com/812552
       results:
         - test: RTCDataChannel-send.html
           status: FAIL
+    - product: chrome
+      url: https://crbug.com/1134607
+      results:
+        - test: RTCDataChannel-binaryType.window.html
+    - product: chrome
+      url: https://crbug.com/1044742
```
Comment (stephenmcgruer) [inline webrtc/META.yml:82]: Unrelated

File: webrtc/META.yml [hunk 4]
```diff
@@ -67,8 +67,37 @@ links:
...
+      results:
+        - test: RTCPeerConnection-createDataChannel.html
+    - product: chrome
+      url: https://crbug.com/884059
```
Comment (stephenmcgruer) [inline webrtc/META.yml:86]: Unrelated

File: webrtc/META.yml [hunk 5]
```diff
@@ -67,8 +67,37 @@ links:
...
+      results:
+        - test: RTCPeerConnection-getStats.https.html
+        - test: RTCPeerConnection-track-stats.https.html
+        - test: RTCRtpReceiver-getStats.https.html
+        - test: RTCRtpSender-getStats.https.html
+    - product: chrome
+      url: https://crbug.com/788737
```
Comment (stephenmcgruer) [inline webrtc/META.yml:93]: Seems poor quality, but technically does mention the test...

**Review by foolip — APPROVED** [review]
Looks good after spot checking.
